tests/test_sunsch.py

```diff
@@ -312,8 +311,8 @@ def test_days_float(readonly_datadir):
     }
     sch = sunsch.process_sch_config(sunschconf)
     assert datetime.datetime(2020, 1, 11, 21, 36, 0) in sch.dates
-    # Rounding is downwards:
-    assert "11 'JAN' 2020/" in str(sch)
+    # TimeVector now support clocktimes
+    assert "11 'JAN' 2020 21:36" in str(sch)
 
 
```

Comment:
issue #440 also tests the `starttime` feature of sunsch. Can  potentially be added, don't remember how important it was. 

Please add something that asserts that microseconds are not leaked into the strings.

Reply:
Thanks, missed starttime - fixed now, with tests. Also ensured microsecs do not appear where not needed.